hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Constants.java

```diff
@@ -1066,4 +1066,29 @@ private Constants() {
    * Require that all S3 access is made through Access Points.
    */
   public static final String AWS_S3_ACCESSPOINT_REQUIRED = "fs.s3a.accesspoint.required";
+
+  /**
+   * What is the smallest reasonable seek that we should group ranges
+   * together during vectored read operation.
+   * Value : {@value}.
+   */
+  public static final String AWS_S3_MIN_SEEK_VECTOR_READS = "fs.s3a.min.seek.vectored.read";
+
+  /**
+   * What is the largest size that we should group ranges
+   * together during vectored read?
```

Comment:
not sure that "readsize" is the best term here

Reply:
we can say merged.size?